Content.Server/GameObjects/Components/NodeContainer/NodeGroups/ApcNetNodeGroup.cs

```diff
@@ -150,9 +150,7 @@ private void PoweredChanged()
 
         private void SetTotalPowerReceiverLoad(int totalPowerReceiverLoad)
         {
-            Debug.Assert(totalPowerReceiverLoad >= 0);
```

Comment:
I don't know, removing this assert doesn't seem like the greatest of ideas. If it was crashing, perhaps it was indicative of a bigger issue?

Reply:
Removed, R wants to look at it.